src/emm/emm_cli.py

```diff
@@ -105,18 +117,8 @@ def git_pull_request(self, args: List[str]) -> None:
 
         :param args: Arguments pass to the github CLI.
         """
-        authenticated = self.modules_service.validate_github_authentication()
```

Comment:
I moved validation to the top-level functions, so we don't need to check it here anymore.